libs/connectors_sdk/connectors_sdk/filtering/validation.py

```diff
@@ -135,6 +135,41 @@ def to_dict(self):
         }
 
 
+class Filter(dict):
```

Comment:
This feels weird in SDK, but there's no easy way to move it out without refactoring